frontend/libs/erxes-ui/src/modules/inputs/components/PhoneField.tsx

```
@@ -359,30 +354,62 @@ const PhoneListFieldRoot = forwardRef<
+      primaryPhone,
+      phones,
+      phoneValidationStatus,
+    );
 
     useEffect(() => {
-      setPhones(phones);
+      setPhones(formattedPhones);
       return () => {
         setShowPhoneInput(false);
       };
       // eslint-disable-next-line react-hooks/exhaustive-deps
```

ESLint

[error] Definition for rule 'react-hooks/exhaustive-deps' was not found.

(react-hooks/exhaustive-deps)